src/lib/holocene/loading.svelte

```diff
@@ -1,19 +1,25 @@
 <script lang="ts">
   import { translate } from '$lib/i18n/translate';
-  import Logo from '$lib/vendor/Temporal_Logo_Animation.gif';
+  import Logo from '$lib/vendor/Temporal_Logo_Animation.mp4';
 
   export let title = translate('common.loading');
 </script>
 
-<div class="my-12 flex flex-col items-center justify-start {$$props.class}">
-  <img
-    src={Logo}
-    style="margin-top: -40px;"
-    alt=""
-    width="200px"
-    height="200px"
-  />
-  <h2 class="text-xl font-medium" style="margin-top: -40px;">
+<div
+  class="my-12 flex flex-col items-center justify-start gap-4 {$$props.class}"
+>
+  <video
+    autoplay
+    loop
+    muted
+    playsinline
+    width="100"
+    height="100"
+    class="dark:invert-[.92]"
```

Comment:
I tried adjusting the filter here to account for the background it is most likely to be on, but not sure if that's really what we want to do here... 

There are still other backgrounds that won't match the background of the mp4 ⬇️ 
|||
|-|-|
|![Screenshot 2024-04-18 at 4 50 53 PM](https://github.com/temporalio/ui/assets/15069288/912aeef8-1096-4008-87f1-58e0006d6436)|![Screenshot 2024-04-18 at 4 51 00 PM](https://github.com/temporalio/ui/assets/15069288/ef88a26f-aa13-49b9-b185-efe51efba3a6)|

Reply:
Any way to get the mp4 with a transparent bg?

Reply:
Unfortunately, no 😞 Need something with an alpha channel.

Reply:
**Update:** Apparently an mp4 **can** support an alpha channel with the H.265 codec. But looks like only in Safari, so added a link to a `.webm` source as well.